metron-interface/metron-config/scripts/start-server-for-e2e.sh

```diff
@@ -14,10 +15,5 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-port: 4201
-
-rest:
-  host: localhost
-  port: 8080
-
-dirPath: alerts-ui
\ No newline at end of file
+SCRIPTS_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$SCRIPTS_ROOT/../node_modules/@angular/cli/bin/ng serve --proxy-config proxy.conf.json --aot $@
```

Comment:
This change was also recognized as a rename by git. It's actually the removal of an unused yml config file and the addition of the same shell script we use in Alert UI to run a local test server.